libbeat/kibana/fields_transformer.go

```diff
@@ -104,13 +98,30 @@ func (t *fieldsTransformer) transformFields(commonFields common.Fields, path str
 	}
 }
 
+func (t *fieldsTransformer) update(target *common.MapStr, override common.Field) error {
+	field, _ := transformField(t.version, override)
+	if override.Type == "" || (*target)["type"] == field["type"] {
+		target.Update(field)
+		return nil
+	}
+	return fmt.Errorf("ERROR: Field <%s> is duplicated. Please update and try again.\n", override.Path)
```

Comment:
@graphaelli I would actually agree here with hound. Why not just remove `ERROR: `? It is already an error by definition.

Reply:
I was just retaining the current behavior.  I assume that was done so that output looks like:

```
ERROR: Field <...> is duplicated. Please update and try again.
```

I'll move the `ERROR:` to the [caller](https://github.com/elastic/beats/blob/4a7e73f01cf06d8fea676c2222376d81d1e7603c/dev-tools/cmd/kibana_index_pattern/kibana_index_pattern.go#L82) and clean this up.